libs/network/esp32/jswrap_esp32_network.c

```diff
@@ -542,23 +548,23 @@ static esp_err_t event_handler(void *ctx, system_event_t *event)
   } // End of handle SYSTEM_EVENT_STA_CONNECTED
 
   if (event->event_id == SYSTEM_EVENT_STA_START) {
-    s_retry_num = 5; // Try to connect 5 times
-      // Perform an esp_wifi_connect
-      esp_err_t err = esp_wifi_connect();
-      if (err != ESP_OK) {
-        jsError( "Wifi: event_handler STA_START: esp_wifi_connect: %d(%s)", err,wifiErrorToString(err));
-        return NULL;
-      }
+    JsVar *jsDetails = jsvNewObject();
```

Comment:
I'm not convinced this is ever used or freed? So it's a memory leak - you probably don't want this

Reply:
Nice, just changed it